src/tests/helpers/scroll-event-data.ts

```diff
@@ -0,0 +1,34 @@
+import { NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
+import { PartialDeep } from 'type-fest';
+
+const contentSize = {
+  height: 200,
+  width: 100
+};
+
+const layoutMeasurement = {
+  height: 100,
+  width: 100
+};
+
+export const scrollDownEventData: PartialDeep<NativeSyntheticEvent<NativeScrollEvent>> = {
+  nativeEvent: {
+    contentOffset: {
+      x: 0,
+      y: contentSize.height
+    },
+    contentSize,
+    layoutMeasurement
+  }
+};
+
+export const scrollUpEventData: PartialDeep<NativeSyntheticEvent<NativeScrollEvent>> = {
+  nativeEvent: {
+    contentOffset: {
+      x: 0,
+      y: -contentSize.height
```

Comment:
@ipakhomov I changed a little bit `scrollUpEventData` to use it like `scrollDownEventData` when items are in reversed order and we can load more by scrolling up

Reply:
@ekazankova thanks! But right now `scrollUpEventData` is unused, let's leave it for future tests